code/datums/chatmessage.dm

```diff
@@ -216,7 +216,7 @@
 	message.appearance_flags = APPEARANCE_UI_IGNORE_ALPHA | KEEP_APART
 	message.alpha = 0
 	message.pixel_y = target.maptext_height
-	message.pixel_x = (target.maptext_width * 0.5) - 16
+	message.pixel_x = -target.base_pixel_x
```

Comment:
@theOOZ looks like you last touched this code to fix an issue in your downstream - from what I could tell, the current code is supposed to just always be zero, but it seems like y'all are doing some stuff differently so let me know if this messes with that at all. I tested it with large mobs and resized mobs and couldn't find any issue